### PR TITLE
xGov-22 Create xGov-Reduce unemployment with NFT Skills Certification.md  -> To withdraw

### PR DESCRIPTION
**Pull request by dikikarim2004**
instead of having to have a college degree, everyone can register their expertise, and have it confirmed by each service user.

### CODE REVIEW

File: Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Reduce unemployment with NFT Skills Certification
+author: Diki karim @dikikarim2010, Diki karim dikikarim2010@gmail.com
+company_name: none
+category: dApps
+focus_area: Education,NFT,Social
+open_source: Yes
+amount_requested: 1000000
+status: Draft
```
Comment (SudoWeezy) [inline Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md:9]: Remember to change this to "Final" if you want your proposal to go through the next voting session.

File: Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Reduce unemployment with NFT Skills Certification
+author: Diki karim @dikikarim2010, Diki karim dikikarim2010@gmail.com
+company_name: none
+category: dApps
+focus_area: Education,NFT,Social
+open_source: Yes
+amount_requested: 1000000
```
Comment (SudoWeezy) [inline Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md:8]: If your project requires multiple milestones, split them into multiple proposals. See: https://github.com/algorandfoundation/ARCs/pull/209

File: Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md (hunk 1 above)
Comment (dikikarim2004) [reply Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md:9]: thank you for your attention

File: Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md (hunk 2 above)
Comment (dikikarim2004) [reply Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md:8]: > If your project requires multiple milestones, split them into multiple proposals. See: [algorandfoundation/ARCs#209](https://github.com/algorandfoundation/ARCs/pull/209)

thank you for your information

**Pull request closed without merging**